mne/event.py

```diff
@@ -299,8 +298,9 @@ def read_events(filename, include=None, exclude=None, mask=None,
     return out
 
 
+@fill_doc
 def write_events(filename, event_list):
```

Comment:
You can do it like this:
```
def write_events(filename, events, *, event_list=None):
    if event_list is not None:
        warn('event_list is deprecated, use "events" instead', DeprecationWarning)
        events = event_list
    del event_list
```

Reply:
Ok, will do, it will be cleaner.

Reply:
Agreed, and I bet 99% of people have done `write_events(fname, my_events)` rather than `write_events(fname, events_list=my_events)` so as far as deprecations go this one should be pretty painless for people